src/bridge/mod.rs

```diff
@@ -182,13 +183,14 @@ async fn start_neovim_runtime(
         close_watcher_running.store(false, Ordering::Relaxed);
     });
 
-    if let Ok(Value::Integer(correct_version)) = nvim.eval("has(\"nvim-0.4\")").await {
-        if correct_version.as_i64() != Some(1) {
-            error!("Neovide requires version 0.4 or higher");
+    if let Ok(output) = nvim.command_output("version").await {
```

Comment:
I'm slightly suspicious of this change. neovim defines options for all of the major versions, so any version greater than 0.4 should cause "has("nvim-0.4")" to return true.

Is this a fix for something I missed? If not, I think I should track down the documentation that indicates this behavior and add it as a comment.

Reply:
While trying to build a snap, I ran into an issue when using the latest neovim version from git. The repo was marked with version 0.5.0, which wasn't caught by has.